technisch_register/webpages.py

```diff
@@ -93,16 +93,42 @@ def create_overview_entry(standard, title_short, description):
     # print standard
     return BS(overview, 'html.parser')
 
-def create_cluster_overview(standards, source, destination_temp, repoCluster, root, assets_path):
-    """Creates a cluster's overview page
+def create_BRT_overview(standards, source, destination_temp, repoCluster, root, assets_path):
```

Comment:
Bedankt voor je feedback, ik ben het helemaal met je eens. Ik heb het
aangepast en een nieuwe pull request gedaan. Wel heb ik ervoor gekozen om
de template toch maar *niet* in een variable aan de functie mee te geven.
Reden daarvoor is dat ik vind dat het inladen van een template onder het
creeeren van webpages valt en wat mij betreft dus in webpages.py thuis
hoort. Het ophalen van de juiste template gebeurt dus nog steeds in de
functie zelf.

De functie voor het creeeren van de cluster overview pages kreeg ook in de
oude situatie al een parameter mee met daarin de cluster naam. Deze wordt
nu in de nieuwe situatie gebruikt om in de functie zelf de juiste template
te openen.

2016-04-12 16:22 GMT+02:00 Simeon Nedkov <notifications@github.com>:

> In technisch_register/webpages.py
> <https://github.com/Geonovum/technisch-register/pull/56#discussion_r59384400>
> :
>
> > @@ -93,16 +93,42 @@ def create_overview_entry(standard, title_short, description):
> >      # print standard
> >      return BS(overview, 'html.parser')
> >
> > -def create_cluster_overview(standards, source, destination_temp, repoCluster, root, assets_path):
> > -    """Creates a cluster's overview page
> > +def create_BRT_overview(standards, source, destination_temp, repoCluster, root, assets_path):
>
> Is het niet overzichtelijker om de te gebruiken template via een parameter
> aan de "oude" functie (create_cluster_overview) te voeren. Nu ontstaan er
> twee functies die nagenoeg hetzelfde doen. Of zie ik iets over het hoofd?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Geonovum/technisch-register/pull/56/files/df9fa2df8b20d784281dfc79dd887cd858780357#r59384400>
>